machine-learning/_posts/2018-08-08-HAMLET.md

```diff
@@ -0,0 +1,213 @@
+   they map the spherical products back to order two, resulting in
+   the feature maps
+
+   $$\mathbf{f}_j(\mathbf{b}_I, \mathbf{b}_{I'}) = \partial^L \circ_2
+                    (g \ast (\mathbf{b}_I \circ_j \mathbf{b}_{I'})$$.
+
+ * **C.** **Linear combination** of *channels*
+
+   Finally, the features of different *channels* are linearly combined
+   to obtain the final filter output $$\mathbf{y}$$ as
+
+   $$\mathbf{y}(D) = \sum_{j, I, I'} \alpha_{jII'} \mathbf{f}_j(\mathbf{b}_I,
+                          \mathbf{b}_{I'})$$.
+
+#### Hierarchical composition
+
+ Since the filters are applied in a multi-resolution approach (see
+ overview figure above) it is necessary to specify how features are
+ forwarded to subsequent scales.
+
+ Given the results on scale *s*, the filter output on scale $$s+1$$ is
+ obtained by
+
+ $$\mathbf{y}^{s+1} := \kappa^s\mathbf{y}^s
+      + |\mathbf{y}^s| \sum_{j,I,I'} \alpha^s_{jII'} \mathbf{f}_j(\mathbf{b}^{s+1}_I,
+                                                                  \mathbf{b}^{s+1}_{I'})
+      + \sum_{j,I,k} \beta^s_{jIk} \mathbf{f}_j(\mathbf{b}^{s+1}_I, \mathbf{y}^s_k)$$,
+
+ which is composed by
+
+  * *a.* the features $$\mathbf{y}^s$$ from the previous scale *s*
+  * *b.* the newly obtained features from scale $$s+1$$
+  * *c.* a mixed term of features $$\mathbf{y}^s$$ from scale *s* and
+         new linear features $$\mathbf{b}^{s+1}_I$$ from scale $$s+1$$.
+
+ **Training**
+
+ The parameters $$\kappa, \alpha$$ and $$\beta$$ are learned during
+ training.
+
+ Given the "trace" of a "tract" the authors define their ground truth
+ as the normalised density map. The unnormlised density map is obtained
+ as
+
+ $$\mathbf{y}(\mathbf{r}) = \sum_k \int \mathbf{Y}^2(\mathbf{n}(t))
+   \delta(\mathbf{r}_k(t) - \mathbf{r})\mathrm{d}t$$,
+
+ where $$\mathbf{r}_k$$ is the parameterisation of a "tract" (?) (check
+ paper for details).
+
+# Experiments
+
+ The experiments are performed on two different data sets. Training
+ is done on a high quality dataset on a Siemens PRISMA scanner
+ (58 directions, b-value $$\in \{1000, 2000}$$ for 55 subjects. Testing
+ (and retesting) uses a low quality data set from a TRIO scanner
+ (60 directions, b-value ***unkown***) on 28 subjects in two sessions.
+
+ **Ground truth** tractography is based on global tractography (check
+ paper for more details). The authors choose 12 bundles to evaluate on.
+
+#### Qualitative assessment on PRISMA test data
+
+ Euler integration is used to obtain streamlines from HAMLET output.
+ Thresholding of the output is performed by "optimising the segmentation
+ accuracy".
+
+ The comparison of tractography output for HAMLET and the ground truth
+ a random test subject from the PRISMA data set is used. The authors
+ find their results overall visually appealing. However, they acknowledge
+ artifacts (yellow circles in the figure below) in the results of their
+ own method. On source of error is high variability in the ground truth
```

Comment:
I don't understand the phrasing of "on source of error is ...."

Reply:
Typo: "One source of ..."

Anyways, I rephrased that sentence a bit.